saferpayofficial.php

```diff
@@ -366,13 +366,11 @@ public function hookDisplayCustomerAccount()
         }
         if (\Invertus\SaferPay\Config\SaferPayConfig::isVersion17()) {
             return $this->context->smarty->fetch(
-                $this->getLocalPath() . 'views/templates/hook/front/MyAccount.tpl'
+                'module:saferpayofficial/views/templates/hook/front/MyAccount.tpl'
             );
         }
 
-        return $this->context->smarty->fetch(
-            $this->getLocalPath() . 'views/templates/hook/front/MyAccount_16.tpl'
-        );
+        return $this->display(__FILE__, 'front/MyAccount_16.tpl');
```

Comment:
not sure this is working correctly, I would prefer keep it in old way, unless tests was done and this allows to override on 1.6 version

Reply:
Smarty display is overridable and path is correct.

Reply:
Method is used in 1.6